index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+    <nav class="navbar">
+        <div class="logo-div">
+            <p class="logo">My Logo</p>
+            <img src="images/Union.png" alt="image" class="nav-image">
+        </div>
+        <div class="navbar-items">
+            <ul>
+                <li><a href="#" class="portfolio">Portfolio</a></li>
+                <li><a href="#" class="about">About</a></li>
+                <li><a href="#" class="contact">Contact</a></li>
+                </ul>
+        </div>
+    </nav>
+</header> 
+    <section class="wrapper">
+        <article class="intro">
+            <h2 class="h2-intro">I'm Eligrand<br/>
```

Comment:
- [ ] As your name is the most important part of this section, it should be wrapped by an `h1` tag, please change it so it applies better semantics documentation on this page. 💯